.eslintrc.js

```diff
@@ -8,21 +8,23 @@ module.exports = {
         node: true,
     },
     globals: {
-        'process': true,
+        'btoa': true,
```

Comment:
Why a global for btoa?

Reply:
it's not actually defining it as global, just the way ESLint works for not throwing undefined errors.  If you use a variable that it does not see defined, ESLint yells at you.  This just tells ESLint that `btoa` is defined in the global scope and not defined anywhere.

Reply:
sorry, didnt even see it was eslintrc -- i assumed it was the webpack config